backend/src/routes/status.py

```diff
@@ -0,0 +1,15 @@
+from os import cpu_count
+
+from fastapi import APIRouter
+from pydantic import BaseModel
+
+router = APIRouter()
+
+class Status(BaseModel):
+    code: int
+    workers: int
+
+
+@router.get("/status", response_model=Status)
```

Comment:
Generally, if we need a ping/healthcheck endpoint (would only be useful if running Docker for `HEALTHCHECK` directives), then we can settle with a very simple ping-pong type of endpoint

```suggestion
@router.get("/ping", include_in_schema=False)
async def ping() -> Response:
    return Response("pong") # HTTP 200 is already the default
```